stores/liker.ts

```diff
@@ -1,40 +1,78 @@
 import { defineStore } from 'pinia'
 
-import { FetchLikerInfoByIdResult } from 'utils/api'
+import { FetchLikerInfoResult } from 'utils/api'
 
 export const useLikerStore = defineStore('liker', () => {
-  const likerUserInfoMap = ref({} as Record<string, FetchLikerInfoByIdResult>)
+  const likerInfoMapById = ref({} as Record<string, FetchLikerInfoResult>)
+  const likerInfoMapByWallet = ref({} as Record<string, FetchLikerInfoResult>)
```

Comment:
Can either of these be a computed value of the other one?

Reply:
How about keeping a map of likerId to wallet?

Reply:
Can, or compute a map from another map using Object.entries and .reduce or sth

Reply:
Done

<img width="842" alt="image" src="https://github.com/user-attachments/assets/47b8ec22-51ab-4704-b4ec-d28ce42e4af4">